crates/goose-cli/src/commands/session.rs

```diff
@@ -81,7 +85,9 @@ pub async fn build_session(
         }
 
         let cmd = parts.remove(0).to_string();
+        let name = parts.last().unwrap().to_string();
```

Comment:
I don't like this @baxen . A ideas?

Reply:
since these are ephemeral, this would work but may have a chance of collisions. i'd instead do generate a random string like we do for sessions here, because it will never be seen and just needs to be a unique entry for the length of the session run

Reply:
will do